src/App.tsx

```diff
@@ -14,11 +14,140 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type SortBy = 'none' | 'name' | 'length';
```

Comment:
let's make it an enum and use in code
like
```suggestion
enum SortBy {
  none,
  name,
  length,
}
```

and use `SortBy.none` etc.